reference/mongodb/bson/unserializable.xml

```diff
@@ -33,14 +33,40 @@
      </ooclass>
     </classsynopsisinfo>
 <!-- }}} -->
-    
+
     <classsynopsisinfo role="comment">&Methods;</classsynopsisinfo>
     <xi:include xpointer="xmlns(db=http://docbook.org/ns/docbook) xpointer(id('class.mongodb-bson-unserializable')/db:refentry/db:refsect1[@role='description']/descendant::db:methodsynopsis[not(@role='procedural')])" />
    </classsynopsis>
 <!-- }}} -->
 
   </section>
 
+  <section role="changelog">
+   &reftitle.changelog;
+   <para>
+    <informaltable>
+     <tgroup cols="2">
+      <thead>
+       <row>
+        <entry>&Version;</entry>
+        <entry>&Description;</entry>
+       </row>
+      </thead>
+      <tbody>
+       <row>
+        <entry>PECL mongodb 1.15.0</entry>
+        <entry>
+         Return types for methods are declared as tentative on PHP 8.0 and newer,
+         triggering deprecation notices in code that implements this interface
+         without declaring the appropriate return types. The <code>#[ReturnTypeWillChange]</code>
```

Comment:
`en/reference/mongodb/bson/unserializable/bsonunserialize.xml` has some example code that would benefit from the addition of `#[ReturnTypeWillChange]`. This would be consistent with what is done in https://www.php.net/manual/en/class.iterator.php and some other interface examples in the PHP.net docs. I believe you can ignore the backslash in the attribute, but I asked about that in https://github.com/php/doc-en/pull/1560#issuecomment-1218446161.

Reply:
I opted for adding return types to the code samples, as the deprecation will require changing at some point. I trust people to make the right choice on whether they can add the return type or have to silence the warning.

Reply:
Ah, adding the return type works just as well. IIRC that wasn't in the original version of this PR.

Reply:
Nope, I pushed that in a subsequent commit after reading your comments :)